service/schemas/user.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+const bCrypt = require("bcryptjs");
+
+const userSchema = new Schema(
+  {
+    password: {
+      type: String,
+      required: [true, "Password is required"],
+    },
+    email: {
+      type: String,
+      required: [true, "Email is required"],
+      unique: true,
+    },
+    subscription: {
+      type: String,
+      enum: ["starter", "pro", "business"],
+      default: "starter",
+    },
+    token: {
+      type: String,
+      default: null,
+    },
+    owner: {
+      type: Schema.Types.ObjectId,
+      ref: "user",
+    },
```

Comment:
Ten klucz powinien być dodany do kontaktów. Bo ty jesteś użytkownikiem który ma kontakty. Ja również loguje się na twoją aplikację i też jestem użytkownikiem ze swoimi kontaktami. W obecnej implementacji jaką masz Ja mam twoje kontakty i ty masz moje. Ten klucz powinien być dodany do schema contacts i kiedy zapisujesz kontakty, zmieniasz je to zawsze od tego momentu powinnaś sprawdzić czy dany kontakt należy do ciebie jeżeli nie pomimo tego że ID się zgadza powinnaś zwrócić nie ma takiego kontaktu. 

Mam nadzieję że ma to sens. :)

Reply:
acha, czyli ma być tylko w contacts?

Reply:
Tak

Reply:
ok